Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM memcached:latest
+
+USER root
+# install python for entrypoint
+RUN apt-get update \
+    && apt-get install -y \
+		netcat \
+    curl
+
+COPY containerbuddy/* /opt/containerbuddy/
+
+# Install Containerbuddy
+# Releases at https://github.com/joyent/containerbuddy/releases
+ENV CONTAINERBUDDY_VERSION 0.1.1
```

Comment:
Use version 1.3; copy it from https://github.com/10up/triton-wordpress/blob/wordpress/Dockerfile#L46-L55